packages/webpack/src/utils/postcss.ts

```diff
@@ -25,7 +26,7 @@ const orderPresets = {
   }
 }
 
-export const getPostcssConfig = (nuxt: Nuxt) => {
+export const getPostcssConfig = async async async async (nuxt: Nuxt) => {
```

Comment:
This seems to be a typo? 🙂 
```suggestion
export const getPostcssConfig = async (nuxt: Nuxt) => {
```

Reply:
thank you! ❤️